src/components/Form/Form.js

```diff
@@ -799,7 +808,10 @@ class Form extends Component {
                                 options.map((option) => (
                                     <MenuItem
                                         key={option}
-                                        value={option}>
+                                        value={option}
+                                        selected={option === "1 Hour" && fieldTitle === "Duration" ? true :
+                                            fieldTitle !== "Duration"}
```

Comment:
Can be replaced with `fieldDuration !== "Duration" || option === "1 hour"` (Is this why we can only select 1 hour options?)

Reply:
lol no, I fixed that but I was trying to do something slick here to show the selected option as 1 hour instead of 0.5 hour